pages/linux/create-image.md

```diff
@@ -0,0 +1,12 @@
+# create-image
+
+> CPIO image creation utility.
+> Commonly found in initramfs-tools packages in some Linux distributions.
+
```

Comment:
```suggestion
> CPIO image creation utility.
> More information: https://git.kernel.org/pub/scm/bluetooth/bluez.git/tree/tools/create-image.c

```
It seems that the information on this command is extremely scarce. For now we'll have to make do with the source code.